include/wlr/xwayland.h

```diff
@@ -212,7 +213,7 @@ void wlr_xwayland_surface_activate(struct wlr_xwayland_surface *surface,
 	bool activated);
 
 void wlr_xwayland_surface_configure(struct wlr_xwayland_surface *surface,
-	int16_t x, int16_t y, uint16_t width, uint16_t height);
+	int16_t x, int16_t y, uint16_t width, uint16_t height, uint32_t mask);
```

Comment:
Maybe make `mask` a `uint16_t` the XCB function takes the mask as `uint16_t`

Reply:
Done.